lib/connectionConfig/connectionConfig.ts

```diff
@@ -80,6 +61,11 @@ export interface ConnectionConfig {
    * connection over WebSockets.
    */
   webSocketEndpointPath?: string;
+  /***
+     * @property {any} [webSocketConstructorOptions] - Options to be passed to the function returned by
+     * rhea.websocket_connect()
```

Comment:
Should this instead be
`Options to be passed to the WebsSocket constructor` ? Especially since the name has "constructor" in it?